tests/ui/reify-intrinsic.stderr

```diff
@@ -2,12 +2,15 @@ error[E0308]: cannot coerce intrinsics to function pointers
   --> $DIR/reify-intrinsic.rs:6:64
    |
 LL |     let _: unsafe extern "rust-intrinsic" fn(isize) -> usize = std::mem::transmute;
-   |            -------------------------------------------------   ^^^^^^^^^^^^^^^^^^^ cannot coerce intrinsics to function pointers
-   |            |
+   |            -------------------------------------------------   ^^^^^^^^^^^^^^^^^^^
+   |            |                                                   |
+   |            |                                                   cannot coerce intrinsics to function pointers
+   |            |                                                   help: consider casting to a fn pointer: `transmute::<_, _> as unsafe extern "rust-intrinsic" fn(isize) -> usize`
```

Comment:
This suggestion doesn't make sense -- can we filter this lint to ignore "rust-intrinsic" functions, which can never be coerced to pointers?

Reply:
I will check